index.html

```diff
@@ -98,10 +98,25 @@ <h1><br>Want to <span>Share</span> or <span>Rent</span> <br>a cab.</h1>
                 <label for="pickupDate">Pickup Date</label>
                 <input type="date" id="pickupDate">
             </div>
+            <script>
+                const today = new Date();
+                const yesterday = new Date(today.setDate(today.getDate())).toISOString().split('T')[0];
+                document.getElementById('pickupDate').setAttribute('min', yesterday);
+              </script>
             <div class="form-group">
                 <label for="dropoffDate">Drop-off Date</label>
                 <input type="date" id="dropoffDate">
             </div>
+            <script>
+                const minDateInput = document.getElementById('pickupDate');
+                const maxDateInput = document.getElementById('dropoffDate');
+                minDateInput.addEventListener('change', function() {
+                    const selectedMinDate = new Date(minDateInput.value);
+                    // Set max date as selected min date 
+                    const maxDate = new Date(selectedMinDate.setDate(selectedMinDate.getDate())).toISOString().split('T')[0];
+                    maxDateInput.setAttribute('min', maxDate); 
+                });
+              </script>
         </div>

```

Comment:
Do not add new script tags, add this script in the tag at the at line 612. 

Also, look at why the css is showing that there are error even after the first load: 

Current Version:
![Screenshot 2024-10-12 at 12 43 21 AM](https://github.com/user-attachments/assets/1077ef95-8b3e-445d-950f-c1d213428b38)

Changes in this PR:
![Screenshot 2024-10-12 at 12 43 26 AM](https://github.com/user-attachments/assets/34e09f49-b275-4f0b-a572-b9f993b34195)